### PR TITLE
Amend wording around introduction of error messages and DEBUG error pages

### PR DESCRIPTION
**Pull request by nimasmi**
Omitted updates from #678

### CODE REVIEW

File: en/django_urls/README.md [hunk 1]
```diff
@@ -111,11 +111,11 @@ As you can see, we're now assigning a `view` called `post_list` to `^$` URL. Thi
 
 The last part `name='post_list'` is the name of the URL that will be used to identify the view. This can be the same as the name of the view but it can also be something completely different. We will be using the named URLs later in the project so it is important to name each URL in the app. We should also try to keep the names of URLs unique and easy to remember.
 
-Everything all right? If you try to visit http://127.0.0.1:8000/ now, then you'll find some sort of 'web page not available' message. It's not running anymore, huh? Take a look at your server console window.
+If you try to visit http:127.0.0.1:8000/ now, then you'll find some sort of 'web page not available' message. This is because the server (remember typing `runserver`?) is no longer running. Take a look at your server console window to find out why.
```
Comment (helenst) [inline en/django_urls/README.md]: the `//` needs to go back in on the URL.

**Comment by helenst** [comment]
This is great, thank you! I like the improvements. Just one small fix needed and then LGTM :+1::confetti_ball:

**Comment by nimasmi** [comment]
Oh yeah. Thanks for spotting!

**Comment by helenst** [comment]
Super, thanks! :palm_tree::sunny:

**Pull request merged**